### ISSUE TITLE
addGroupCoverages error[hdf5 file accessibility problem]....!

### ISSUE THREAD

**Issue by R-Krait** [GreenleafLab/ArchR#1183]
I want to execute addGroupCoverages function. It used to work fine but suddenly the error occurred. As I think, Maybe the permission of .h5 file was denied during writing group coverage .h5 files. However, even if my thoughts about the cause of this error are right, I cannot think of a suitable solution......!

So I need help. What should I do in this situation?

My error message is as follows:
ArchR logging to : ArchRLogs/ArchR-addGroupCoverages-14811b49f96dc7-Date-2021-11-25_Time-16-36-27.log
If there is an issue, please report to github with logFile!
2021-11-25 16:36:27 : Creating Coverage Files!, 0.013 mins elapsed.
2021-11-25 16:36:27 : Batch Execution w/ safelapply!, 0.013 mins elapsed.
Error in (function (..., threads = 1, preschedule = FALSE)  : 
Error Found Iteration 1 : 
	[1] "Error in h(simpleError(msg, call)) : \n  error in evaluating the argument 'values' in selecting a method for function 'Rle': HDF5. File accessibility. Unable to open file.\n"
	<simpleError in h(simpleError(msg, call)): error in evaluating the argument 'values' in selecting a method for function 'Rle': HDF5. File accessibility. Unable to open file.>
Error Found Iteration 2 : 
	[1] "Error in h(simpleError(msg, call)) : \n  error in evaluating the argument 'values' in selecting a method for function 'Rle': HDF5. File accessibility. Unable to open file.\n"
	<simpleError in h(simpleError(msg, call)): error in evaluating the argument 'values' in selecting a method for function 'Rle': HDF5. File accessibility. Unable to open file.>
Error Found Iteration 3 : 
	[1] "Error in h(simpleError(msg, call)) : \n  error in evaluating the argument 'values' in selecting a method for function 'Rle': HDF5. File accessibility. Unable to open file.\n"
	<simpleError 
In addition: Warning message:
In mclapply(..., mc.cores = threads, mc.preschedule = preschedule) :
  15 function calls resulted in an error

My ArchR log file was uploaded in this bug report:
[ArchR-addGroupCoverages-14811b49f96dc7-Date-2021-11-25_Time-16-36-27.log](https://github.com/GreenleafLab/ArchR/files/7601056/ArchR-addGroupCoverages-14811b49f96dc7-Date-2021-11-25_Time-16-36-27.log)

**Comment by R-Krait** [comment]
Ah! I solved it by changing the default permission setting in .bashrc file and restarting R studio....

**Comment by rcorces** [comment]
@R-Krait - can you clarify what you mean by changing the default permission setting in your .bashrc? This issue comes up sporadically for other users as well so perhaps your solution might help them.

**Comment by R-Krait** [comment]
@rcorces  I wrote the 'umask 000' in my .bashrc file. And then, I ran the 'addGroupCoverages' again and it worked properly.

**Comment by wanghlv** [comment]
> @rcorces I changed the global permission setting by using the 'umask 000' in my .bashrc file. And then, I ran the 'addGroupCoverages' again and it worked properly.

Thank you for this solution!! it fixed my problems as well!!
